eng/pipelines/prettier.yml

```diff
@@ -0,0 +1,14 @@
+trigger: none
+
+jobs:
+- job: Prettier
```

Comment:
OpenAPI Prettier ?

Reply:
We can align this with whatever we call the pipeline but currently the pipeline is where the check name comes from.